src/YoloDev.Expecto.TestSdk/execution.fs

```diff
@@ -101,12 +102,12 @@ module private PrinterAdapter =
       result.Duration <- duration
       recordEnd result
 
-    { Expecto.Impl.TestPrinters.silent with
-        beforeEach = beforeEach
-        passed = passed
-        ignored = ignored
-        failed = failed
-        exn = exn }
+    Expecto.Impl.TestPrinters.silent
```

Comment:
This could also use the new alias, no?

Reply:
I waffled on that. I felt like leaving the entry point fully qualified made it a bit clearer that it's operating on an Expecto type. I don't have strong feelings about it though. I'd be down to use the alias

Reply:
I prefer the initial and the resulting chain's `TestPrinters` to be the same, and in this case, since it's repeated many times, using an alias is preferred.

Reply:
Done